src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,73 @@
-export const Pagination = () => {};
+import React from 'react';
+import { getNumbers } from '../../utils';
+import cn from 'classnames';
+
+interface Props {
+  perPage: number;
+  total: number;
+  currentPage: number;
+  onPageChange: (page: number) => void;
+}
+
+export const Pagination: React.FC<Props> = ({
+  perPage,
+  total,
+  currentPage,
+  onPageChange,
+}) => {
+  const pagesList: number[] = getNumbers(1, Math.ceil(total / perPage));
```

Comment:
Consider memoizing the 'pagesList' calculation to avoid unnecessary recalculations on each render when the 'perPage', 'total', or 'currentPage' props do not change.

Reply:
done